### PR TITLE
new suite - unit5x

### PR DESCRIPTION
**Pull request by sergeysviridenko**
Hello!

* Type: new feature
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/incubator/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change: new suits for php 5.x tests

Thanks

### CODE REVIEW

File: tests/unit5x/Mvc/MongoCollectionTest.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Phalcon\Test\Mvc;
+namespace Phalcon\Test\unit5x\Mvc;
```
Comment (sergeyklay) [inline tests/unit5x/Mvc/MongoCollectionTest.php]: `Phalcon\Test\unit5x\Mvc` => `Phalcon\Test\Unit5x\Mvc`

File: tests/unit5x/Validation/Validator/MongoIdTest.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Phalcon\Test\Validation\Validator;
+namespace Phalcon\Test\unit5x\Validation\Validator;
```
Comment (sergeyklay) [inline tests/unit5x/Validation/Validator/MongoIdTest.php]: `Phalcon\Test\unit5x\Validation\Validator` =>  `Phalcon\Test\Unit5x\Validation\Validator`

File: composer.json [hunk 3]
```diff
@@ -52,6 +52,7 @@
         "psr-4": {
             "Phalcon\\Test\\": "tests/unit/",
             "Phalcon\\Test\\Aerospike\\": "tests/aerospike/",
+            "Phalcon\\Test\\unit5x\\": "tests/unit5x/",
```
Comment (sergeyklay) [inline composer.json]: `"Phalcon\\Test\\unit5x\\": "tests/unit5x/",` => `"Phalcon\\Test\\Unit5x\\": "tests/unit5x/",`

File: .travis.yml [hunk 4]
```diff
@@ -84,6 +84,7 @@ script:
   - vendor/bin/codecept build
   - vendor/bin/codecept run -v tests/unit
   - '[[ "$PHP_MAJOR" == "7" ]] || vendor/bin/codecept run -v tests/aerospike'
+  - '[[ "$PHP_MAJOR" == "7" ]] || vendor/bin/codecept run -v tests/unit5x'
```
Comment (Jurigag) [inline .travis.yml]: Well, shouldn't this unit name be unit7x if we check php_major version against 7?

Comment (sergeyklay) [inline .travis.yml]: it is **OR**: `<condition> OR <condition>`:

```
- (is it php 7) OR (run php 5 related tests)
```

Comment (Jurigag) [inline .travis.yml]: But those tests are php 7 related.

Comment (sergeyklay) [inline .travis.yml]: which ones?

Comment (Jurigag) [inline .travis.yml]: `tests/unit5x`

Comment (sergeyklay) [inline .travis.yml]: The `tests/unit5x` tests uses PECL `mongo` extension. This extension is deprecated and no longer supported in PHP 7

Comment (Jurigag) [inline .travis.yml]: It's not true, ti's uses mongodb extension.

Comment (sergeyklay) [inline .travis.yml]: Hm, yes `CollectionsTest` moved by mistake. But `MongoIdTest` uses mongo extension: https://github.com/phalcon/incubator/blob/3.2.x/tests/unit5x/Validation/Validator/MongoIdTest.php#L53

Comment (sergeyklay) [inline .travis.yml]: @sergeysviridenko  Could you please roll back `CollectionsTest`?

**Pull request merged**